### DISCUSSION TITLE
Is there any official Telegram group or channel of Sketchware pro?

### DISCUSSION THREAD

**Discussion by TheRealAshik** [Sketchware-Pro/Sketchware-Pro#1489] [General]
Is there any official Telegram group or channel of Sketchware Pro?

**Comment by PranavPurwar** [comment]
https://t.me/sketchwarepro

**Reply by TheRealAshik** [reply]
> https://t.me/sketchwarepro

![Screenshot_20241209_171009_Telegram.png](https://github.com/user-attachments/assets/57cc7e6f-ccf1-4d2c-9c68-b9630f9cb466)

The link is not working. That's why I asked the community.
